OJS.Workers.ExecutionStrategies/Extensions/ExecutionContextExtensions.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace OJS.Workers.ExecutionStrategies.Extensions
+{
+    using System;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+
+    using Ionic.Zip;
+
+    using OJS.Workers.Common;
+
+    public static class ExecutionContextExtensions
+    {
+        public static void SanitizeContent(this ExecutionContext executionContext)
+        {
+            var callingClassName = new StackFrame(1).GetMethod()?.DeclaringType?.Name;
```

Comment:
Can't you instead use the [CallerMemberName attribute here](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.callermembernameattribute?view=netframework-4.7.2) ?